pkg/kubectl/cmd/apply.go

```diff
@@ -69,6 +69,7 @@ func NewCmdApply(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	usage := "Filename, directory, or URL to file that contains the configuration to apply"
 	kubectl.AddJsonFilenameFlag(cmd, &options.Filenames, usage)
 	cmd.MarkFlagRequired("filename")
+	cmd.Flags().Bool("overwrite", true, "Ignore conflicts between modified and current configuration")
```

Comment:
Please clarify what ignoring means in this help text.  Maybe `Automatically resolve conflicts between the modified and live configurations by using values from the modified configuration.`